database/README.md

```diff
@@ -7,148 +7,98 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/README.
 
-1.  `dirty_background_ratio`, by default `10`.
-2.  `dirty_ratio`, by default `20`.
+Since Netdata Agents usually run on the edge, inside production systems, Netdata Agent **parents** should be considered.
+When having a **parent - child** setup, the child (the Netdata Agent running on a production system) delegates all its
+functions, including longer metrics retention and querying, to the parent node that can dedicate more resources to this
+task. A single Netdata Agent parent can centralize multiple children Netdata Agents (dozens, hundreds, or even thousands
+depending on its available resources).
 
-These control the amount of memory that should be dirty for disk syncing to be triggered. On dedicated Netdata servers,
-you can use: `80` and `90` respectively, so that all RAM is given to Netdata.
+## Running Netdata in embedded devices
 
-With these settings, you can expect a little `iowait` spike once every 10 minutes and in case of system crash, data on
-disk will be up to 10 minutes old.
+Embedded devices usually have very limited RAM resources available.
 
-![image](https://cloud.githubusercontent.com/assets/2662304/23835030/ba4bf506-0768-11e7-9bc6-3b23e080c69f.png)
+There are 2 settings for you to tweak:
 
-To have these settings automatically applied on boot, create the file `/etc/sysctl.d/netdata-memory.conf` with these
-contents:
+1. `[db].update every`, which controls the data collection frequency
+2. `[db].retention`, which controls the size of the database in memory (except for `[db].mode = dbengine`)
 
-```conf
-vm.dirty_expire_centisecs = 60000
-vm.dirty_background_ratio = 80
-vm.dirty_ratio = 90
-vm.dirty_writeback_centisecs = 0
-```
+By default `[db].update every = 1` and `[db].retention = 3600`. This gives you an hour of data with per second updates.
 
-There is another mode to help overcome the memory size problem. What is **most interesting for this setup** is
-`[db].mode = dbengine`.
+If you set `[db].update every = 2` and `[db].retention = 1800`, you will still have an hour of data, but collected once
+every 2 seconds. This will **cut in half** both CPU and RAM resources consumed by Netdata. Of course experiment a bit.
+On very weak devices you might have to use `[db].update every = 5` and `[db].retention = 720` (still 1 hour of data, but
+1/5 of the CPU and RAM resources).
 
-### dbengine
+You can also disable [data collection plugins](/collectors/README.md) you don't need. Disabling such plugins will also
+free both CPU and RAM resources.
 
-In this mode, the database of Netdata is stored in database files. The [Database Engine](/database/engine/README.md)
-works like a traditional database. There is some amount of RAM dedicated to data caching and indexing and the rest of
-the data reside compressed on disk. The number of history entries is not fixed in this case, but depends on the
-configured disk space and the effective compression ratio of the data stored.
+## Running a dedicated parent Netdata server
```

Comment:
Not sure why need a section on this in this document. References in the previous mentions of parents or streaming will be sufficient.

Reply:
I will omit it